modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/featureflag/Feature.kt

```diff
@@ -219,6 +219,14 @@ enum class Feature(
         hasFirebaseRemoteFlag = true,
         hasDevToggle = true,
     ),
+    APPSFLYER_ANALYTICS(
+        key = "appsflyer_analytics",
+        title = "AppsFlyer Analytics",
+        defaultValue = BuildConfig.DEBUG,
+        tier = FeatureTier.Free,
+        hasFirebaseRemoteFlag = false,
```

Comment:
What is the shipping version for this feature? If it has to be `7.86`, does it need to be set to true?

Reply:
Good point, thanks. I will change it.